### DISCUSSION TITLE
Hardware

### DISCUSSION THREAD

**Discussion by pseddonxvi** [nebhead/garage-zero-v2#1] [Q&A] [answered]
After getting V1 to work and playing around with it for a bit, I figured V2 had a few more features that probably suit my needs better. Since it's mentioned that the custom board isn't necessary for this version in the documentation, I was wondering if any modifications are needed to upgrade V1 hardware for V2, along with all the wiring/GPIOs for the extra reed switches/second door.

**Comment by nebhead** [comment, marked as answer]
Yep - After you install, you'll want to go into settings, configure the garage door settings and change the GPIO settings:

![image](https://github.com/nebhead/garage-zero-v2/assets/19876886/587afc4a-1f26-4a27-baa9-fb73980eaa60)

If you have a standard v1 setup, then: 

**On the sensors side:** 
Delete the Limit Sensor Open switch. 
Change the Limit Sensor Closed Switch to GPIO18
Uncheck the Sensor Trigger Level High box

**On the buttons side:** 
Change the door button to GPIO15
Ensure that Button/Relay Trigger Level High is checked

That should work just fine.

**Reply by pseddonxvi** [reply]
Awesome, thanks!
